File: app/src/main/java/com/example/samson/immvrse/OnboardingFragmentFour.java
package com.example.samson.immvrse;

import android.content.Intent;
import android.databinding.DataBindingUtil;
import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.example.samson.immvrse.databinding.FragmentOnboardingFourBinding;

public class OnboardingFragmentFour extends Fragment {

    private String title;
    private int page;

    public static OnboardingFragmentFour newInstance(int page, String title) {
        OnboardingFragmentFour fragment = new OnboardingFragmentFour();
        Bundle bundle = new Bundle();
        bundle.putInt("page", page);
        bundle.putString("title", title);
        fragment.setArguments(bundle);
        return fragment;
    }

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        page = getArguments().getInt("page", 0);
        title = getArguments().getString("title");
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        FragmentOnboardingFourBinding binding = DataBindingUtil.inflate(inflater, R.layout.fragment_onboarding_four, container, false);
        binding.getRoot();
        binding.continueButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(getActivity(), LocationActivity.class);
                startActivity(intent);
            }
        });
return binding.getRoot();
    }
    }

File: app/src/main/java/com/example/samson/immvrse/OnboardingFragmentFour.kt
package com.example.samson.immvrse

import android.content.Intent
import android.databinding.DataBindingUtil
import android.os.Bundle
import android.support.v4.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

import com.example.samson.immvrse.databinding.FragmentOnboardingFourBinding

class OnboardingFragmentFour : Fragment() {

    private var title: String? = null
    private var page: Int = 0

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        page = arguments!!.getInt("page", 0)
        title = arguments!!.getString("title")
    }

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        val binding = DataBindingUtil.inflate<FragmentOnboardingFourBinding>(inflater, R.layout.fragment_onboarding_four, container, false)
        binding.root
        binding.continueButton.setOnClickListener { view ->
            val intent = Intent(activity, LocationActivity::class.java)
            startActivity(intent)
        }
        return binding.root
    }

    companion object {

        fun newInstance(page: Int, title: String): OnboardingFragmentFour {
            val fragment = OnboardingFragmentFour()
            val bundle = Bundle()
            bundle.putInt("page", page)
            bundle.putString("title", title)
            fragment.arguments = bundle
            return fragment
        }
    }
}
